From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 11 Jun 2014 10:23:16 +0200
Subject: tmpfiles: don't allow read access to journal files to users not in
 systemd-journal

(cherry picked from commit 176f2acf8dee45fee832fd2ab07243f63783a238)
---
 tmpfiles.d/systemd.conf | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/tmpfiles.d/systemd.conf b/tmpfiles.d/systemd.conf
index c470045..79911d9 100644
--- a/tmpfiles.d/systemd.conf
+++ b/tmpfiles.d/systemd.conf
@@ -22,7 +22,10 @@ d /run/systemd/users 0755 root root -
 d /run/systemd/machines 0755 root root -
 d /run/systemd/shutdown 0755 root root -
 
-m /var/log/journal 2755 root systemd-journal - -
-Z /var/log/journal/%m 2755 root systemd-journal - -
-m /run/log/journal 2755 root systemd-journal - -
-Z /run/log/journal/%m 2755 root systemd-journal - -
+z /run/log/journal 2755 root systemd-journal - -
+z /run/log/journal/%m 2755 root systemd-journal - -
+z /run/log/journal/%m/*.journal 0640 root systemd-journal - -
+
+z /var/log/journal 2755 root systemd-journal - -
+z /var/log/journal/%m 2755 root systemd-journal - -
+z /var/log/journal/%m/*.journal 0640 root systemd-journal - -
